views/layouts/default.tpl

```diff
@@ -5,16 +5,23 @@
   <title>Google Scraper</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  <link type="text/css" rel="stylesheet" href="static/css/application.css" />
-  <link type="text/css" rel="stylesheet" href="static/css/tailwind.css" />
+  {{ assets_css "css/application.css" }}
+  {{ assets_css "css/tailwind.css" }}
```

Comment:
1. Is `assets_css` from Beego?
2. Why is Tailwind imported separately and below the application styles?